src/projects/navigation/project-pull-request-list-item.html

```diff
@@ -81,16 +102,25 @@
       </div>
       <div class="main">
         <list-item-label type="title">
-        <a href$="{{_cistatus.target_url}}" target="_blank"><iron-icon title="CI passed" class$="icon-passed [[_cistatus.state]]" icon="icons:check-circle"></iron-icon></a>
-        <a href$="{{_cistatus.target_url}}" target="_blank"><iron-icon title="CI failed" class$="icon-failed [[_cistatus.state]]" icon="icons:cancel"></iron-icon></a>
+          <span class="ci-placeholder" hidden$="[[_hasCIStatus]]"></span>
+          <span hidden$="[[!_hasCIStatus]]">
+            <a href$="{{_cistatus.target_url}}" target="_blank">
+              <iron-icon title="CI passed" class$="icon-passed [[_cistatus.state]]" icon="icons:check-circle"></iron-icon>
+            </a>
+            <a href$="{{_cistatus.target_url}}" target="_blank">
+              <iron-icon title="CI failed" class$="icon-failed [[_cistatus.state]]" icon="icons:cancel"></iron-icon>
+            </a>
+          </span>
           <span title$="[[pr.title]]">[[pr.title]]</span>
         </list-item-label>
         <list-item-label type="subtitle">[[pr.user.login]]</list-item-label>
-        <list-item-label type="box" title="Status" hidden$="[[_hasLabels(showStatus, issue)]]">[[_getLabelNames(issue.labels)]]</list-item-label>
+        <span class="labels">
+          <list-item-label type="box" title="Status" hidden$="[[_shouldHideLabels]]">[[_getLabelNames(issue.labels)]]</list-item-label>
```

Comment:
Is it not possible to give the `list-item` a height? This would save an extra span.